domain/PointsProg.php

```diff
@@ -5,78 +5,139 @@
 class PointsProg {
     private $id;
     private $name;
-    private $abbrevName;
-    private $date;
-    private $startTime;
-    private $endTime;
-    private $description;
-    private $location;
-    private $capacity;
-    private $trainingMedia;
-    private $postMedia;
-    private $animalId;
-
-    function __construct($id, $name, $abbrevName, $date, $startTime, $endTime, $description, $location, $capacity, $trainingMedia, $postMedia, $animalId) {
+    private $address;
+    private $freezer_meals;
+    private $allergies;
+    private $snacks;
+    private $snack_notes;
+    private $foodlion;
+    private $giant;
+    private $walmart;
+    private $wegmans;
+    private $sheetz;
+    private $wawa;
+    private $house_cleaning;
+    private $lawn_care;
+    private $AAA_membership;
```

Comment:
variable names should not be capitalized. find some kind of consistancy between naming variables, either camel case or underscores.

Reply:
Good catch! I will be sure to change the name of the AAA variables to be lowercase everywhere.  I think as a team, we decided we aren't necessarily going to be super picky about sticking with either camel case or underscores; it is okay to have either since both are used in the base code. I will likely stick to underscores since that is what I prefer, but you are right about ensuring there is no capitalization at the beginning of the variable name.